roles/nfs_volumes/library/partitionpool.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+"""
+Ansible module for partitioning.
```

Comment:
Have you submitted this upstream? We'll still need to carry it (at least until an upstream version that includes it is set at the minimum supported version), but it would be nice to limit the number of generic modules that we carry eventually.